packages/data-stores/src/plans/constants.ts

```diff
@@ -19,7 +19,7 @@ interface Currency {
 
 export const plansOrder = [ PLAN_PERSONAL, PLAN_PREMIUM, PLAN_BUSINESS, PLAN_ECOMMERCE ];
 
-export const plansPaths = [ 'beginner', 'personal', 'premium', 'business', 'ecommerce' ];
+export const plansPaths = [ 'free', 'personal', 'premium', 'business', 'ecommerce' ];
```

Comment:
For the sake of safety is it worth creating a type for these paths so it minimises the risk of one of them being misspelt in future, or the addition of an invalid path going into this array?